README.md

```diff
@@ -34,14 +34,25 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+Adaptive layouts and responsive layouts both use media queries but the difference is that adaptive layouts use fixed layouts widths and responsive uses responsive units. 
+
+
 2. Describe what it means to be mobile first vs desktop first.
 
+Mobile first and Desktop first really only describe in which direction the development of a site is started. mobile first starts designing in reference to mobile sizes and adds or takes away as the screen sizes increase to larger platforms like tablet then to desktop. this uses @media queries to increase appearnce and function at those different screen sizes. Making the page adaptive or responsive at all sizes.
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
+The 62.5% covers the size of a readers screen and adjusts the content font size to make it easier for the fonts to be adjusted throught the code to make the page responsive to the zoom in and zoom out features a patron may need to view the design better.
+
```

Comment:
It also gives us a base of 10px on our root element, which makes it easier to calculate rems.